### PR TITLE
Add script to convert MIOpen driver to ckProfiler

### PR DESCRIPTION
**Pull request by bartekxk**

### CODE REVIEW

**Review by illsilin — APPROVED** [review]

File: profiler/include/profiler/profile_grouped_conv_bwd_weight_impl.hpp [hunk 1]
```diff
@@ -167,99 +168,115 @@ bool profile_grouped_conv_bwd_weight_impl(int do_verification,
...
+                        std::cout << "Fail info: " << op_ptr->GetTypeString() << std::endl;
+                    }
+
+                    all_pass &= pass;
+
+                    if(do_log)
+                    {
+                        LogRangeAsType<float>(std::cout << "output : ", output.mData, ",")
+                            << std::endl;
+                        ;
```
Comment (aosewski) [inline profiler/include/profiler/profile_grouped_conv_bwd_weight_impl.hpp:253]: Empty line with just semicolon.

File: profiler/include/profiler/profile_grouped_conv_bwd_weight_impl.hpp [hunk 2]
```diff
@@ -167,99 +168,115 @@ bool profile_grouped_conv_bwd_weight_impl(int do_verification,
...
+                        LogRangeAsType<float>(
+                            std::cout << "weight (device): ", weight_device_result.mData, ",")
+                            << std::endl;
+                        ;
```
Comment (aosewski) [inline profiler/include/profiler/profile_grouped_conv_bwd_weight_impl.hpp:257]: Empty line with just semicolon.

File: profiler/include/profiler/profile_grouped_conv_bwd_weight_impl.hpp [hunk 3]
```diff
@@ -167,99 +168,115 @@ bool profile_grouped_conv_bwd_weight_impl(int do_verification,
...
+                        LogRangeAsType<float>(
+                            std::cout << "weight (host): ", weight_host_result.mData, ",")
+                            << std::endl;
+                        ;
```
Comment (aosewski) [inline profiler/include/profiler/profile_grouped_conv_bwd_weight_impl.hpp:261]: Empty line with just semicolon.

File: profiler/include/profiler/profile_grouped_conv_bwd_weight_impl.hpp [hunk 4]
```diff
@@ -167,99 +168,115 @@ bool profile_grouped_conv_bwd_weight_impl(int do_verification,
...
+                        LogRangeAsType<float>(std::cout << "input: ", input.mData, ",")
+                            << std::endl;
+                        ;
```
Comment (aosewski) [inline profiler/include/profiler/profile_grouped_conv_bwd_weight_impl.hpp:264]: Empty line with just semicolon.

File: script/convert_miopen_driver_to_profiler.py [hunk 5]
```diff
@@ -0,0 +1,386 @@
...
+def run_ck_profiler_cmd(cmd):
+    print("ckProfiler command:")
+    print(cmd)
+    subprocess.run(cmd)
+
+
+def parse_data_type(args):
+    if args.data_type == "fp32":
+        if args.ck_profier_op == "grouped_conv_bwd_weight" or \
+           args.ck_profier_op == "grouped_conv_bwd_weight" or \
```
Comment (aosewski) [inline script/convert_miopen_driver_to_profiler.py:29]: Duplicated line. You probably meant to have bwd_weight and bwd_data.

**Review by aosewski — COMMENTED** [review]
There are few minor things and one bug in this PR. Please fix it.

**Pull request merged**